### DISCUSSION TITLE
Initial Setup option for preview max settings

### DISCUSSION THREAD

**Discussion by foxhoundv** [nextcloud/all-in-one#548] [Questions] [answered]
I have it setup where people can upload from their phone, photos to the Nextcloud. I came across an issue a while back that took me quite some time to figure out how to correct. Because more and more phones are taking higher resolution and better pictures, they wouldn't display properly because the max preview size was never set in the config. I had to add 'preview max memory' => 1024 and 'preview max x' => 2048, and 'preview max y' => 2048.

Is there a way, or even would this be something others want, to add a closing setup page during install that would allow the admin to add apps and change config settings if needed upon finalization of install?

**Comment by szaimen** [comment, marked as answer]
Hello, we will not add a GUI option to change either of these options. If you want to change it youself you can do like you've already done.

Regarding the preview max memory, there is already a discussion happening here: https://github.com/nextcloud/server/issues/31284. We will use whatever default comes out in the end.

**Reply by foxhoundv** [reply]
Ok, thanks for the input.
